### PR TITLE
LPS-133848 Add condition to display unpublished tab on kaleo forms admin

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:forward

**Comment by rafaprax** [comment]
ci:stop

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/workflow.jsp [hunk 1]
```diff
@@ -120,15 +120,17 @@ if (tabs1.equals("published")) {
 
 		<aui:nav-item href="<%= viewPublishedURL %>" label="published" selected='<%= tabs1.equals("published") %>' />
```
Comment (rafaprax) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-web/src/main/resources/META-INF/resources/admin/process/workflow.jsp:121]: Hey @seltong, 
```java
<aui:nav-item href="<%= viewPublishedURL %>" label="published" selected='<%= tabs1.equals("published") %>' />
````
must be into the if statement as well.

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by rafaprax** [comment]
Just started reviewing :)

**Comment by seltong** [comment]
Reopened in https://github.com/liferay-workflow/liferay-portal/pull/405

**Pull request closed without merging**